Assets/Samples/RebindingUI/RebindActionUI.cs

```diff
@@ -268,14 +268,16 @@ void CleanUp()
                     operation =>
                     {
                         m_RebindStopEvent?.Invoke(this, operation);
-                        m_RebindOverlay?.SetActive(false);
```

Comment:
Why this change - its the same thing isn't it.

Reply:
Changes suggested by a user: https://forum.unity.com/threads/rebinding-ui-sample-bugs-fixes.1307430/

"The Elvis operator doesn't work with UnityEngine.Object types in some circumstances. It seems to be a long standing issue, see [this thread](https://forum.unity.com/threads/elvis-operator-null-coalescing-operator.433767/) for details."